pkg/util/ready/ready.go

```diff
@@ -100,6 +101,27 @@ func CheckDeploymentIsReady(ctx context.Context, cli appsv1client.DeploymentInte
 	}
 }
 
+// CheckAROOperatorDeploymentIsReady returns a function which polls AROOperator Deployment and
+// returns its readiness
+func CheckAROOperatorDeploymentIsReady(ctx context.Context, cli appsv1client.DeploymentInterface, name string) func() (bool, error) {
+        return func() (bool, error) {
+                d, err := cli.Get(ctx, name, metav1.GetOptions{})
+                switch {
+                case kerrors.IsNotFound(err):
+                        return false, nil
+                case err != nil:
+                        return false, err
+                }
+
+		if d.Labels["version"] == version.GitCommit {
+			return DeploymentIsReady(d), nil
+		} else {
+			return nil
+		}
+        }
```

Comment:
The formatting is odd here. Have you installed the Go plugin for vscode? It has an option to autoformat on save, which should correct this.